cmd/rokka/cli/massupload/massuploader.go

```diff
@@ -0,0 +1,13 @@
+package massupload
```

Comment:
I'd prefer to have that in one file in the CLI package instead. It's not that much code.

Reply:
Done, moved everything into the cmd_massupload.go file